matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -36,6 +36,37 @@ use uuid::Uuid;
 
 type Packet = Box<[u8]>;
 
+#[derive(Debug)]
+/// General configuration options for a WebRtc connection
+pub struct WebRtcSocketConfig {
+    pub room_url: String,
+    /// Configuration for the (single) ICE server
+    pub ice_server: RtcIceServerConfig
+}
+
+#[derive(Debug)]
+/// Configuration options for an ICE server connection.
+/// See also: https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer#example
+pub struct RtcIceServerConfig {
+    /// An ICE server instance can have several URLs. Use an empty Vec to disable ICE
```

Comment:
```suggestion
    /// An ICE server instance can have several URLs.
```
Yep, probably best to just remove it, and will deal with disabling/multiple in a separate pr.

Reply:
Interesting that GitHub shows it like this but I did that exact change already before your change request: https://github.com/johanhelsing/matchbox/pull/28/commits/f623ce7bb4c85261f3c5203572f2b5fc534de8be